### ISSUE TITLE
Limit text line length in default page/article views

### ISSUE THREAD

**Issue by quamsta** [uiowa/uiowa#2924]
# Problem
While testing some migration patterns, looking at SiteNow websites on campus and just through general CMS usage, I've found that when viewing a page on larger screens, body text on pages without additional layout builder cards or containers is allowed to span then entire width of the layout__spacing_container element,  which is around 1310 "pixels."

This presents general user experience, readability, and accessibility issues because the recommended maximum number of characters per line is anywhere between **50 to 75** characters (depending on the source) and in some cases on some SiteNow sites, I've found the number of characters per line to be around **160**. 

Sources:

http://webtypography.net/2.1.2
https://baymard.com/blog/line-length-readability

<img width="1424" alt="Screen Shot 2021-01-25 at 2 16 13 PM" src="https://user-images.githubusercontent.com/472923/105762718-600a2c80-5f1a-11eb-972f-7efdbaebaab0.png">

# Steps to reproduce

Create a new page, fill out the content field, view it in a non-mobile/wider display.

# Proposed solution

Create a max-width property for the page body text and limit it to around 45em (roughly 720px), possibly centering the body and title divs (mocked up below).

I understand there's a lot going on here and this solution may present other issues around laying out blocks and other bits of content, but I feel this is important for text-heavy sites and publications on campus to maintain readability.

![Screen Shot 2021-01-25 at 2 39 29 PM](https://user-images.githubusercontent.com/472923/105763198-238b0080-5f1b-11eb-9595-b7b566f96002.png)

**Comment by bspeare** [comment]
@quamsta Here's a related issue for Articles: https://github.com/uiowa/uiowa/issues/2813#issuecomment-767109876.   I'm not sure I'd want to see a centered layout because our header design is left-aligned.

**Comment by quamsta** [comment]
@bspeare If there's a sidebar with additional info, like in that Article issue you linked, I think that would be ideal, but if there's no sidebar and it's a very basic page with text, I don't see the issue with putting the body text in the center of a large container. The headings are all still left-aligned. Is the problem that the site title/unit name isn't lined up with the body text?

Text-heavy sites like Medium seem to do well with a left-aligned header and centered text:
https://blog.medium.com/medium-acquires-social-ebook-platform-glose-cf2e94485d8

Either way, I feel automatically limiting the number of characters per line on text-heavy pages like this should be explored in some fashion. My eyes get tired reading those long lines ! :)

**Comment by bspeare** [comment]
@quamsta I think if the centered is the default site-wide layout like with that medium blog, the medium example can make sense.  But when the site template jumps around from left aligned to centered, I think it is a more jarring user experience.  But even your medium example adds a sidebar once you scroll:

<img width="1646" alt="Screen Shot 2021-01-25 at 4 32 21 PM" src="https://user-images.githubusercontent.com/1036433/105774578-078f5b00-5f2b-11eb-8e05-8d8e85ec14b6.png">

**Comment by bspeare** [comment]
Just adding a few links for future reference.  These sites do a nice job of setting the default centered width for the entire site: https://advanced.jhu.edu/academics/, https://www.msmc.edu/admissions-aid/military-admissions/, And an example of a left-aligned narrow column for the entire site: https://engineering.jhu.edu/partner-with-us/.

**Comment by joewhitsitt** [comment]
Can we move the guts of this issue over to https://github.com/uiowa/uiowa/issues/2784 where it is already pointed?

**Comment by quamsta** [comment]
@joewhitsitt Totally fine to mark this as a duplicate as that issue (is that something you all do?) I didn't see that original issue, which is similar. I'll comment on the original issue too.

**Comment by joewhitsitt** [comment]
@quamsta thanks! trying to keep the backlog manageable by consolidating where possible. We usually just say "closing in favor of #X" which links to and back.

**Comment by quamsta** [comment]
Duplicate of #2784

**Comment by quamsta** [comment]
Closing in favor of #2784

**Comment by mjoneill** [comment]
Is there a slice through this that would be suitable for an accessibility sprint?
